### PR TITLE
Disable pop-up accent window on OS X

### LINKED ISSUE #1137: Press and hold keys on OSX causes suggestions popup
- **OS version and name**: OSX 10.12.1
- **Hyper.app version**: 1.0.0.1303
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: NA
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue
Just updated to 1.0.0, now when I'm in neovim if i press and hold down the 'L' key instead of repeating the movements, it also flickers the popup to enter 'ł'. It also seems to affect all keys as pressing and holding jkli keys seems sluggish. Was fine before this and is still fine in iterm. See attached gif which captures a subset of the popups flashing

press hold neovim

### PR DESCRIPTION
**Pull request by roolebo**
Before:
flickering-popus

After:
no-popups

Fix #1137

### CODE REVIEW

**Comment by matheuss** [comment]
@roolebo for now I'll leave this as `blocked` because I'd like to fix this – I had it working while implementing the `hyper-caret`, as you can see in this gif i recorded back last year:
gif

**Comment by chabou** [comment]
It would be great to make this accent keyboard working and maybe let user optionaly disabling it.
But for now, it is better to disabling it like iTerm.

File: app/index.js [hunk 1]
```diff
@@ -215,6 +215,10 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       } else {
         console.log('ignoring auto updates during dev');
       }
+      // Disable accent pop-up
+      if (process.platform === 'darwin') {
+        systemPreferences.setUserDefault('ApplePressAndHoldEnabled', 'boolean', false);
```
Comment (chabou) [inline app/index.js]: It is not the right place to do this. `setUserDefault`should be called only once and not at every window init.
Can you fix merge conflicts and move your block directly in `app.on('ready',...)` callback?

**Comment by chabou** [comment]
This is not relevant anymore.
BTW thank you so much for your help and really sorry for answer delays 🙏

**Pull request closed without merging**